exoscale/instances.go

```diff
@@ -97,13 +96,13 @@ func (i *instances) NodeAddressesByProviderID(ctx context.Context, providerID st
 		return nil, err
 	}
 
-	instanceName := *instance.Name
+	instanceName := instance.Name
```

Comment:
i think we can remove this variable as it's not a simple copy and just use instance.Name

Reply:
https://github.com/exoscale/exoscale-cloud-controller-manager/pull/97/commits/cf1bcfe84897dd7a7db300d20e084252d6152b72